examples/blog/invoiceGenerator/src/components/pdf/pdfLayout.tsx

```diff
@@ -12,12 +12,14 @@ type PdfProps = {
 };
 
 export const PdfLayout: React.FC<PdfProps> = ({ record }) => {
+    // eslint-disable-next-line
     const pdfExportComponent = useRef<any>();
```

Comment:
```suggestion
    const pdfExportComponent = useRef<PDFExport>(null);
```

Looks like that package already comes with type definitions 😅

Reply:
done